spec/models/tenant_spec.rb

```diff
@@ -856,4 +856,21 @@
       expect(projects.map(&:first)).to eq(%w(root.proj3 root.ten1.proj1 root.ten2.proj2))
     end
   end
+  describe "builds tenant tree" do
```

Comment:
newline and in describe should be name of method what you describing  the `.` is for instance methods and `#` for class methods so our case:
`describe ".build_tenant_tree"`